x-pack/plugins/security_solution/common/detection_engine/schemas/types/threat_mapping.test.ts

```diff
@@ -173,4 +175,48 @@ describe('threat_mapping', () => {
     ]);
     expect(message.schema).toEqual({});
   });
+
+  test('it should not validate concurrent_searches if it is < 0', () => {
```

Comment:
@rylnd had pointed this out in my tests before, that `should not validate` is somewhat ambiguous as to whether its validation should not be performed or validation should fail. It tripped me up for a quick second too so thought I'd just note it.

Reply:
Oh I see. That's a good point. I will change this and during my next code changes try to be clearer and just say something like:

```ts
it should fail validation when concurrent_searches is < 0
```

for everyone

Reply:
Changed that one and the rest in this file.